test/src/go.mod

```diff
@@ -1,11 +1,10 @@
 module github.com/cloudposse/terraform-example-module
 
-go 1.17
+go 1.18
```

Comment:
We might as well update this in `github.com/cloudposse/terraform-example-module` too but that's obviously outside of this pr